supportedBackends/google/pipelines/v2beta/src/test/scala/cromwell/backend/google/pipelines/v2beta/api/DeserializationSpec.scala

```diff
@@ -31,7 +31,7 @@ class DeserializationSpec extends FlatSpec with Matchers {
               "actionId" -> Integer.valueOf(18),
               "ipAddress" -> "86.127.54.8",
               "portMappings" -> Map(
-                8000 -> 8008
+                "8000" -> Integer.valueOf(8008)
```

Comment:
Presumably because real `json` doesn't allow you to have integer keys in objects?

If so, this should be fine as long as real-life operations metadata doesn't come back with integer keys in pseudo-json here!